### PR TITLE
Missena Bid Adapter : send coppa and autoplay

### PR DESCRIPTION
**Pull request by pdamoc**
## Type of change

- [x] Updated bidder adapter   

## Description of change

The change adds support for `coppa` and a test for the capability to autoplay videos.

### CODE REVIEW

File: modules/missenaBidAdapter.js [hunk 1]
```diff
@@ -91,8 +92,10 @@ function toPayload(bidRequest, bidderRequest) {
   const bidFloor = getFloor(bidRequest);
   payload.floor = bidFloor?.floor;
   payload.floor_currency = bidFloor?.currency;
-  payload.currency = config.getConfig('currency.adServerCurrency') || 'EUR';
+  payload.currency = config.getConfig('currency.adServerCurrency');
   payload.schain = bidRequest.schain;
+  payload.coppa = config.getConfig('coppa') === true ? 1 : 0;
```
Comment (ChrisHuie) [inline modules/missenaBidAdapter.js:97]: can you please get coppa from `bidderRequest.ortb2.regs.coppa` instead?

Comment (pdamoc) [reply modules/missenaBidAdapter.js]: I have updated the code to get the `coppa` from the `bidderRequest`.

Comment (pdamoc) [reply modules/missenaBidAdapter.js]: @ChrisHuie is there anything else that we need to do here?

**Review by ChrisHuie — APPROVED** [review]

File: modules/missenaBidAdapter.js (hunk 1 above)
Comment (ChrisHuie) [reply modules/missenaBidAdapter.js]: looks good to me

**Pull request merged**